packages/plugin-commands-installation/src/install.ts

```diff
@@ -124,6 +124,10 @@ export function help () {
             description: `The directory in which the ${WANTED_LOCKFILE} of the package will be created. Several projects may share a single lockfile`,
```

Comment:
As per our convention, there should be a period at the end of the second sentence.